pkg/services/ngalert/eval/evaluator_mock.go

```diff
@@ -51,10 +51,10 @@ type FakeEvaluator_ConditionEval_Call struct {
 }
 
 // ConditionEval is a helper method to define mock.On call
```

Comment:
This change is out of scope with the issue.

Reply:
@matyax 
done, removed!
thanks for the review!

Reply:
Thank you! 

If you don't mind, I'd suggest you to try shorter and more descriptive commit messages. For example:
- `Rename file to follow conventions` or `Elasticsearch: rename file to follow conventions`
- `Revert unwanted change` or `Revert <previous commit name>`.